tests/model_fields/test_imagefield.py

```diff
@@ -32,6 +33,9 @@ class Person:
     PersonWithHeight = PersonWithHeightAndWidth = PersonDimensionsFirst = Person
     PersonTwoImages = Person
 
+    class NoReadImageModel:
+        pass
+
```

Comment:
```suggestion
    PersonTwoImages = NoReadImageModel = Person
```
I think we can do something like this